FlowCrypt/Functionality/Services/Local Private Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -38,7 +38,7 @@ final class InMemoryPassPhraseStorage: PassPhraseStorageType {
         passPhraseProvider.remove(passPhrases: [passPhrase])
     }
 
-    func getPassPhrases() -> [PassPhrase] {
+    func getPassPhrases(for email: String) -> [PassPhrase] {
         passPhraseProvider.passPhrases
```

Comment:
For `InMemoryPassPhraseStorage`, email field is not respected.
This also applies to `logOutUser` too.
To check/delete passphrase from user email in `InMemoryPassPhraseStorage` we need to pass KeyService or AppContext to get key pairs from user email first(And then get matching pass phrases from there).
But this seems big changes and need to be addressed in another issue I think.
@tomholub Please confirm.
 
![image](https://user-images.githubusercontent.com/77344191/165946279-3f7ed9ae-52ad-463f-b833-c50198ae27e8.png)

Reply:
That's a real mess. I'll file another issue for afterwards. Too much in this PR.

Reply:
filed https://github.com/FlowCrypt/flowcrypt-ios/issues/1551